<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScreen</class>
 <widget class="QDialog" name="RegisterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">RegisterScreen {
    background-color: #f0f0f0;
    color: #333333;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 13px;
}

QGroupBox {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 5px;
    font-size: 16px;
    font-weight: bold;
    color: #333333;
    margin-top: 10px;
    padding: 20px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    left: 10px;
}

QLabel {
    font-size: 14px;
    color: #333333;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
    min-height: 35px;
}

QLineEdit:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QPushButton {
    background-color: #70B9FE;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 14px;
    min-height: 30px;
    min-width: 90px;
}

QPushButton:hover {
    background-color: #5aa9fe;
}

QPushButton:pressed {
    background-color: #4a99fe;
}</string>
  </property>
  <widget class="QGroupBox" name="UserRegGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>831</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>User Registration</string>
   </property>
   <widget class="QLineEdit" name="usernameField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="contactField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>UserID</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="contactLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Contact</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confPasswordField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="confPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>120</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adwaita Mono</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>120</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adwaita Mono</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fullnameField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>281</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fullnameLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Full Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
